clang/lib/CIR/CodeGen/CIRGenExpr.cpp

```diff
@@ -698,8 +698,10 @@ static LValue buildGlobalVarDeclLValue(CIRGenFunction &CGF, const Expr *E,
 
   auto V = CGF.CGM.getAddrOfGlobalVar(VD);
   auto RealVarTy = CGF.getTypes().convertTypeForMem(VD->getType());
-  // TODO(cir): do we need this for CIR?
```

Comment:
@bcardosolopes  This is  the comment I was talking about)

Reply:
Traditional LLVM codegen doesn't contain this anymore, so it's probably not relevant. While looking at it though, can you take the chance and add a missing:
```
  if (VD->getTLSKind() != VarDecl::TLS_None)
    llvm_unrecheable("NYI");
```